sdcm/test_config.py

```diff
@@ -233,3 +249,8 @@ def set_ip_ssh_connections(cls, ip_type):
     @classmethod
     def set_duration(cls, duration):
         cls.TEST_DURATION = duration
+
+    @classmethod
+    def set_rsyslog_imjournal_rate_limit(cls, interval: int, burst: int) -> None:
+        cls.RSYSLOG_IMJOURNAL_RATE_LIMIT_INTERVAL = interval
+        cls.RSYSLOG_IMJOURNAL_RATE_LIMIT_BURST = burst
```

Comment:
Not having these be properties I don't see any need in this method, because in such case these values can be redefined anytime anywhere not using this method.

Reply:
Just followed the logic of `TestConfig` (ex. `Setup`) class. Don't see any reason to do it in another way right now.

Reply:
The reason was mentioned in my comment. Amount of code won't differ a lot, but it's safeness will be bigger.
We are not forced to follow the "existing" coding approach.

Reply:
@bentsi @dkropachev do you have a decision what to do with all that stuff like `TEST_DURATION`, etc. in this class? Maybe it's time to make the next turn?

Reply:
> @bentsi @dkropachev do you have a decision what to do with all that stuff like `TEST_DURATION`, etc. in this class? Maybe it's time to make the next turn?

I think we need to get rid of them one by one, not sure it is a time to start it.
In this particular ocasion you canjust pass parameters to `get_startup_script` , but i'am also fine if you do it old way

Reply:
I thought about parameters for `get_startup_script()` first, but it requires to change it in too many places.

Reply:
@vponomaryov agree, created a task to refactor TestConfig and convert the class methods to properties